primitives/src/v1.rs

```diff
@@ -798,6 +798,10 @@ sp_api::decl_runtime_apis! {
 		// Additionally, some runtime APIs may depend on state that is pruned on the `on_initialize`.
 		// At the moment of writing, this is `candidate_events`.
 
+		/// Get the next, current and some previous authority keys deduplicated.
+		#[skip_initialize_block]
+		fn authorities() -> Vec<AuthorityDiscoveryId>;
```

Comment:
What's the intended difference here with `AuthorityDiscoveryApi::authorities()`?